src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -162,6 +184,23 @@ public void verifyInstructorEditFormDisabled(int instrNum) {
         assertTrue(isNotEditable);
     }
 
+    public void verifyInstructorChangesAfterEdit(int instrNum, String newName, String newEmail,
+            Boolean newIsDisplayedToStudents, String newDisplayName, String newRole) {
+        assertEquals(newName, getInstructorName(instrNum));
+        if (!newEmail.isEmpty()) {
+            assertEquals(newEmail, getInstructorEmail(instrNum));
+        }
```

Comment:
The problem with that arises from the test case where we are making changes to a yet-to-join instructor. Here, the email field is readonly and so if I try to edit it, the test throws an error. Hence I used this and passed a blank string when editing the email of a yet-to-join instructor.